addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/WakeOnLanUtility.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.samsungtv.internal;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.net.InterfaceAddress;
+import java.net.NetworkInterface;
+import java.util.Enumeration;
+
+import org.openhab.binding.samsungtv.internal.handler.SamsungTvHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class with utility functions to support Wake On Lan (WOL)
+ *
+ * @author Arjan Mels - Initial contribution
+ *
+ */
+public class WakeOnLanUtility {
+
+    private static Logger logger = LoggerFactory.getLogger(SamsungTvHandler.class);
+
+    /**
+     * Get MAC address for host
+     * uses "arping" tool
+     *
+     * @param hostName Host Name (or IP address) of host to retrieve MAC address for
+     * @return MAC address
+     */
+    public static String getMACAddress(String hostName) {
+        try {
+            Process proc = Runtime.getRuntime().exec("arping -r -c 1 -C 1 " + hostName);
+            proc.waitFor();
+            BufferedReader stdInput = new BufferedReader(new InputStreamReader(proc.getInputStream()));
+            String macAddress = stdInput.readLine();
+            logger.info("MAC address of host {} is {}", hostName, macAddress);
```

Comment:
this should not log info, only debug

Reply:
Autodetection is only done once in the binding, so seems to make sense to log with info as user might want to look this up, isn't it? (Although result will also be visible in the configuration.)